sys/net/include/ieee802154_frame.h

```diff
@@ -8,19 +9,20 @@
 
 /**
  * @defgroup    net_ieee802154 IEEE802.15.4
- * @ingroup net
- * @brief   IEEE802.15.4 adapaption layer
+ * @ingroup     net
+ * @brief       IEEE802.15.4 adapaption layer
  * @{
  *
  * @file
- * @brief   IEEE 802.14.4 framing data structs and prototypes
+ * @brief       IEEE 802.14.4 framing data structs and prototypes
  *
- * @author  Stephan Zeisberg <zeisberg@mi.fu-berlin.de>
- * @author  Oliver Hahm <oliver.hahm@inria.fr>
+ * @author      Stephan Zeisberg <zeisberg@mi.fu-berlin.de>
+ * @author      Oliver Hahm <oliver.hahm@inria.fr>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
  */
 
-#ifndef IEEE802154_IEEE802154_FRAME
-#define IEEE802154_IEEE802154_FRAME
+#ifndef __IEEE802154_FRAME_H
+#define __IEEE802154_FRAME_H
```

Comment:
Leading underscores are not really recommended in include guard macro names.

http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf 

>7.1.3 Reserved identifiers
>All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use.